### PR TITLE
Added a note about using decimal-object on project level.

### PR DESCRIPTION
**Pull request by limeeugenia**

### CODE REVIEW

File: 202001.0/5f4778e0-8c57-4959-b3b4-a8c37f477f3b.md [hunk 1]
```diff
@@ -24,9 +24,108 @@ APPLICATION_STORE=US console propel:schema:copy
 console propel:install
 console transfer:generate
 ```
+4. Update the code that uses the product availabilities API response to handle the numeric strings. The API response changed in the following way:
```
Comment (PhilinTv) [inline 202001.0/5f4778e0-8c57-4959-b3b4-a8c37f477f3b.md:27]: Please mention that this appeared on 4.1.0 release. 
Also please update the composer require above, as the first valid v4 came only at 4.1.

**Review by PhilinTv — CHANGES_REQUESTED** [review]
-

File: 202001.0/5f4778e0-8c57-4959-b3b4-a8c37f477f3b.md (hunk 1 above)
Comment (PhilinTv) [inline 202001.0/5f4778e0-8c57-4959-b3b4-a8c37f477f3b.md:27]:
```suggestion
4. Update apps and integrations that use product availabilities API. From now on it requires the support of high precision availability quantity in string representation (15 will become '15.0000000000'). The API response changed in the following way: 
```

Comment (PhilinTv) [reply 202001.0/5f4778e0-8c57-4959-b3b4-a8c37f477f3b.md:27]: Also we need to mark 4.0.0 as a dead release, nobody should migrate there.

**Review by tamasnyulas — APPROVED** [review]

**Review by PhilinTv — APPROVED** [review]

File: 202001.0/5f4778e0-8c57-4959-b3b4-a8c37f477f3b.md (hunk 1 above)
Comment (lenadoc) [reply 202001.0/5f4778e0-8c57-4959-b3b4-a8c37f477f3b.md]: The PR has been merged manually, because of the technical issue.

**Pull request closed without merging**